src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/CollectionSettingsV0Reader.java

```diff
@@ -37,8 +40,10 @@ public CollectionSettings readCollectionSettings(
    * readCollectionSettings method based on interface method signature
    */
   @Override
-  public CollectionSettings readCollectionSettings(
-      JsonNode jsonNode, String collectionName, ObjectMapper objectMapper) {
+  public CollectionSchemaObject readCollectionSettings(
+      JsonNode jsonNode, String keyspaceName, String collectionName, ObjectMapper objectMapper) {
+    // TODO: this is really confusing, why does this implement the interface and not implement the
```

Comment:
100% agree, this is confusing and makes no sense. Probably worth filing a follow-up issue.

Reply:
Leave a TODO here